awx/api/views.py

```diff
@@ -3719,16 +3731,26 @@ def update(self, request, *args, **kwargs):
         return super(JobDetail, self).update(request, *args, **kwargs)
 
 
-class JobExtraCredentialsList(SubListAPIView):
+class JobCredentialsList(SubListAPIView):
 
     model = Credential
     serializer_class = CredentialSerializer
     parent_model = Job
-    relationship = 'extra_credentials'
+    relationship = 'credentials'
     new_in_320 = True
     new_in_api_v2 = True
 
 
+class JobExtraCredentialsList(JobCredentialsList):
```

Comment:
if you subclass this to a new concrete view, it won't really be `new_in_320`

Reply:
👍 good catch